lively.morphic/tooltips.cp.js

```diff
@@ -1,28 +1,61 @@
 import { component } from './components/core.js';
 import { pt, Color } from 'lively.graphics';
-import { HorizontalLayout, Label } from 'lively.morphic';
+import { TilingLayout } from './layout.js';
+import { Label, ViewModel } from 'lively.morphic';
```

Comment:
Do not import from the `index.js`. Instead import from the modules these entities are declared in directly.

Reply:
Perfect, that works! Thanks!